docs/content/basic-concepts.md

```diff
@@ -106,18 +114,18 @@ To create a Spectacle presentation that lives in a single HTML page, you will on
   // add some content
   const deck = html`
     <${Deck} theme=${customTheme}>
-        <${Slide}>
-            <${FlexBox} height="100%" flexDirection="column">
-              <${Heading} fontSize="150px">SPECTACLE</Heading>
-              <${Heading} fontSize="h2">A ReactJS Presentation Library</Heading>
-            <//>
-          <//>
+      <${Slide}>
+        <${FlexBox} height="100%" flexDirection="column">
+          <${Heading} fontSize="150px">SPECTACLE</Heading>
```

Comment:
Should we do the `<//>` instead of the JSX closing tag?

Reply:
Yep! A related note, I'm looking at follow on work for htm@3 which I think has real close tags!